coalib/coala.py

```diff
@@ -59,11 +60,34 @@ def main(debug=False):
 
         if args.show_bears:
             from coalib.settings.ConfigurationGathering import (
-                get_filtered_bears)
-
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, log_printer)
-
+                get_all_bears)
+            filtered_bears = get_all_bears(log_printer)
+            if args.filter_by_language:
+                # filter_by_language option should be deprecated soon.
+                # Since it is already covered by filter_by option.
+                log_printer.warn(
+                    "'--filter-by-language ...' option is deprecated. "
+                    "Use '--filter-by language ...' option instead.")
+                args.filter_by = [] if args.filter_by is None else \
+                    args.filter_by
```

Comment:
Continuation of https://github.com/coala/coala/pull/4240#discussion_r122601529

In this solution the `else` part is obsolete. Then it would be better to keep:

```python
if args.filter_by is None:
    args.filter_by = []
```

But IMO it's still sufficient enough to just do:

```python
args.filter_by = args.filter_by or []
```

If it's already an empty `list` it doesnt' really hurt ;)